### ISSUE TITLE
The tab menu always hides after closing a tab

### ISSUE THREAD

**Issue by moray33** [anthonycr/Lightning-Browser#225]
After closing a tab, the "tab menu" hides automatically. This way, if you want to close several tabs at once, it's really annoying because you have to open the menu time after time. I think it shouldn't hide autimatically after closing a tab.

**Comment by anthonycr** [comment]
I agree. However, pro-tip: if you close tabs that you don't have open and save the open tab until last, it won't close on you haha. I will remove the closing functionality though.

**Comment by epitron** [comment]
This looks like a duplicate of [#276](https://github.com/anthonycr/Lightning-Browser/issues/276)
